debug/leaflet-include.js

```diff
@@ -37,7 +37,12 @@
 
 	var path = getSrcUrl();
     for (var i = 0; i < scripts.length; i++) {
-		document.writeln("<script src='" + path + scripts[i] + "'></script>");
+		if (scripts[i].substr(-3) === '.js') {
+			document.writeln("<script src='" + path + scripts[i] + "'></script>");
+		}
+		if (scripts[i].substr(-4) === '.css') {

```

Comment:
Great work, will help a lot navigating the CSS files. Maybe rename `scripts` here to `assets` or `sources` now they are not scripts?